docs/installation/manage/_index.md

```diff
@@ -260,6 +260,49 @@ cat $(brew --prefix)/var/log/viam.log
 {{% /tab %}}
 {{< /tabs >}}
 
+## Uninstall `viam-server`
+
+{{< tabs name="Uninstall viam-server">}}
+{{% tab name="Linux"%}}
+
+Remove the system installed service with the following three commands:
+
+```sh {class="command-line" data-prompt="$"}
+sudo systemctl disable --now viam-server
+sudo rm /etc/systemd/system/viam-server.service
+sudo systemctl daemon-reload
+```
+
```

Comment:
Forgot that we also tell people to install the config file at /etc/viam.json As such, also need

To remove the configuration file, run:
sudo rm /etc/viam.json

Reply:
Done